frontend/src/components/messstelle/charts/BelastungsplanMessquerschnittCard.vue

```diff
@@ -490,9 +482,9 @@ function drawMessstelleInfo() {
 function drawLegende() {
     const formeln = new Map<string, string>();
     formeln.set("KFZ", "KFZ = Pkw + Lkw + Lz + Bus + Krad");
-    formeln.set("SV", "SV = Lkw + Lz + Bus");
+    formeln.set("(SV)", "SV = Lkw + Lz + Bus");
     formeln.set("GV", "GV = Lkw + Lz");
-    formeln.set("SV%", "SV-Anteil = SV : KFZ x 100(%)");
+    formeln.set("(SV%)", "SV-Anteil = SV : KFZ x 100(%)");
     formeln.set("GV%", "GV-Anteil = GV : KFZ x 100(%)");
     formeln.set("Rad", "");
```

Comment:
Die Klammer ist nicht nur einfach bei SV oder SV% sondern bei dem zweiten Element in der Anzeige. Sprich:
Element 1 (Element 2) Element (3)

Reply:
hä, warum? Warum will man bei jedem zweiten element ne klammer haben?

Reply:
Nicht per se bei jedem zweiten. Im Belastungsplan kannst du dir ja insgesamt 3 von möglichen 5 Werten (KFZ, SV, GV, SV% und GV% ) anzeigen lassen. Damit das dann besser lesbar ist wird um den jeweiligen zweiten Wert eine Klammer gesetzt. Das kannst du nicht hardcoden, sondern muss berechnet werden. Je nachdem was alles ausgewählt ist die Reihenfolge immer so: KFZ > SV > GV > SV% > GV%

Beispiele: 
![image](https://github.com/it-at-m/dave-frontend/assets/18263366/ca8333be-2416-4474-b130-220fcc560265)
![image](https://github.com/it-at-m/dave-frontend/assets/18263366/8e1ab2bd-92dd-4c0f-9861-33174d767b98)
![image](https://github.com/it-at-m/dave-frontend/assets/18263366/231c7776-5c0b-4237-be11-a89fd8963aaf)